mne/viz/evoked.py

```diff
@@ -1753,8 +1778,31 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     _setup_ax_spines(axes, vlines, tmin, tmax, invert_y, ymax_bound, unit,
                      truncate_xaxis)
 
-    if len(conditions) > 1:
-        plt.legend(loc='best', ncol=1 + (len(conditions) // 5), frameon=True)
+    if show_sensors:
+        try:
```

Comment:
can you avoid the try?

Reply:
I'm not sure how I could do anything more elegant in this case. Isn't this just what try/except is made for?

Reply:
looks okay to me

Reply:
ok fair enough